static/js/portico/help.js

```diff
@@ -17,10 +17,42 @@ function registerCodeSection($codeSection) {
     $li.eq(0).click();
 }
 
+function adjust_mac_shortcuts() {
+    var keys_map = new Map([
+        ['Backspace', 'Delete'],
+        ['Enter', 'Return'],
+        ['Home', 'Fn + ⇽'],
+        ['End', 'Fn + ⇾'],
+        ['PgUp', 'Fn + ↑'],
+        ['PgDn', 'Fn + ↓'],
+    ]);
+
+    $(".markdown .content code").each(function () {
+        var text = $(this).text();
+
```

Comment:
at least on the backend, we use less spacing (fewer newlines), so that more stuff fits on a screen. Not sure what the coding style is on the frontend though.

Reply:
Are you asking me to remove the new lines between the blocks? The new line spacing is consistent with the rest of the file, I think.